src/campaign/sass/components/_cta-faa.scss

```diff
@@ -0,0 +1,11 @@
+.cta-faa{
+  background-color: $apprentice;
+  color: #FFFFFF;
+  display: flex;
+  align-items: center;
+  padding: 30px;
+}
+
+.cta-faa__heading{
+  color: #FFFFFF !important;
```

Comment:
Shouldn't need !important

Reply:
agreed, same issue as last time. had a look and its because the heading styles include the colour which is overriding the styles defined above